index.html

```diff
@@ -253,8 +362,8 @@ <h2>About me</h2>
     <section>
       <div class="form-container">
         <p>
-          I'm always <br> interested in hearing about new projects,
-          so if you'd like to chat please get in.
+          I'm always interested <br> in hearing about new <br> projects,
+          so if you'd <br> like to chat please <br> get in touch.
         </p>
         <form action="https://formspree.io/f/xeqwonbn" method="POST">
           <ul>
```

Comment:
- [ ] Kindly 😃try to fix the margin and padding of the form and between elements of the form 
- [ ] Also consider aligning the form at the center because I see that you aligned it on the left side 💯
- [ ] Also try to add the first name and last name inputs to be like the Figma Design as I see that you change the two inputs to be one input👌
- [ ] Also kindly😊 try to put the button submit at the correct position in the form like the Figma Design 
- [ ] Please as stated by the previous reviewer try to fix the font size of the text ```i am always...``` to be like the Figma Design💯

> Check the table below to see the difference 😊⤵️

| Figma design | Your design| 
| --- | --- |
| ![image](https://user-images.githubusercontent.com/83360636/214831772-fc2b3c03-4297-4b0f-ae68-e789c674004c.png)|  ![image](https://user-images.githubusercontent.com/83360636/214831826-048ecbdd-2e7b-41a9-8bb3-0fec5bcff309.png)|

Reply:
@osamaashraf6 please see this screenshot 

![image](https://user-images.githubusercontent.com/118761303/214839381-aebbd2d1-8eee-4782-9523-41bf3edc923a.png)

> * [x]  Also try to add the first name and last name inputs to be like the Figma Design as I see that you change the two inputs to be one input👌

Reply:
> @osamaashraf6 please see this screenshot
> 
> ![image](https://user-images.githubusercontent.com/118761303/214839381-aebbd2d1-8eee-4782-9523-41bf3edc923a.png)
> 
> > * [x]   Also try to add the first name and last name inputs to be like the Figma Design as I see that you change the two inputs to be one input👌

- Okay @AbdusSattar-70 I highlight 😃that to you because you put in the last review two inputs after that you put one, but I am re-review, so when I see that I edited the review, it can be one input or two check up😊please